### PR TITLE
[#35] Remove redundancy in Travis CI build

### LINKED ISSUE #35: Remove redundancy in Travis CI build
Look here: https://github.com/paypal/NNAnalytics/blob/a73dbc6a5a3dce96fdec027a28e91bdf6789cd1a/.travis.yml#L18

It turns out we only need to run `clean build` arguments as `build` itself _already_ covers license, findbugs, checkstyle, and tests.

You can verify this locally by just running `./gradlew clean build buildRpm`.

Please note that we still need to run `buildRpm`.

### PR DESCRIPTION
**Pull request by pauljv92**
Make sure you have checked all steps below.

### GitHub Issue
Fixes #35 

### Checklist:

- [] This pull request updates the documentation
- [x] This pull request changes the code
- [x] Title of the PR is of format (example) : `[#25] Add Pull Request Template`, where `[#25] is the GitHub issue number`

### Commit Guidelines
- [x] My commits all reference GH issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,4 +15,4 @@ matrix:
     - env: VERSION='-PhadoopVersion=2.9.0'
     - env: VERSION='-PhadoopVersion=3.0.0'
 script:
-  - ./gradlew clean build licenseMain licenseTest findbugsMain findbugsTest buildRpm $VERSION
```
Comment (zero45) [inline .travis.yml]: Oh man. Sorry, @pauljv92, the $VERSION is definitely still needed as well.

Comment (pauljv92) [reply .travis.yml]: gotchu, will fix it

**Comment by pauljv92** [comment]
@zero45 changes made and validated

**Comment by zero45** [comment]
Thank you, @pauljv92.

**Pull request merged**